webui/react/src/pages/F_ExpList/CompareParallelCoordinates.tsx

```diff
@@ -265,6 +266,10 @@ const CompareParallelCoordinates: React.FC<Props> = ({
     return <Spinner center spinning />;
   }
 
+  if (trials.length === 0) {
+    return <Message title="No data to plot." type={MessageType.Empty} />;
+  }
+
```

Comment:
As https://hpe-aiatscale.atlassian.net/browse/WEB-1482 says, show `No data to plot` when trial count is 0